src/actions.rs

```diff
@@ -477,16 +477,25 @@ impl ActionHandler {
 
         let mut buf = Vec::<u8>::new();
         match format_input(input, &config, Some(&mut buf)) {
-            Ok(_) => {
-                // Note that we don't need to keep the VFS up to date, the client
-                // echos back the change to us.
-                let range = ls_util::range_from_vfs_file(&self.vfs, &path);
-                let text = String::from_utf8(buf).unwrap();
-                let result = [TextEdit {
-                    range: range,
-                    new_text: text,
-                }];
-                out.success(id, ResponseData::TextEdit(result))
+            Ok((summary, ..)) => {
+                // It's workaround because currently rustfmt returns ok even if there are parsing errors.
```

Comment:
This is not really a workaround because it is intentional behaviour from rustfmt - it doesn't return an Error because there was no error in rustfmt (while this is kind of contentious, a parsing error is a variety of expected behaviour from rustfmt's perspective because it has caught and handled the error)